wicker/core/storage.py

```diff
@@ -92,17 +125,24 @@ def fetch_file_s3(self, input_path: str, local_prefix: str, timeout_seconds: int
         os.makedirs(os.path.dirname(local_path), exist_ok=True)
 
         while not os.path.isfile(success_marker):
-            with SimpleUnixFileLock(lock_path, timeout_seconds=timeout_seconds):
+            with SimpleUnixFileLock(lock_path, timeout_seconds=1000):
```

Comment:
@isaak-willett I had a chat with @aalavian where he mentioned we should not be doing this. Can we change the timeout back here?